workshop/exercise-4/README.md

```diff
@@ -2,37 +2,24 @@
 
 ### Challenges with microservices
 
-For the longest time in the history of app development, apps were built with monolith mindset. Monolith apps have a large number of instances running all of the services provided in one app. Things like user account management, payment, and reporting are all run from a shared resource. This worked pretty well until service-oriented architecture (SOA) came along and promised us a much brighter future. The basic principle of SOA is to break down apps to smaller components, and having them to talk to one other using protocols like REST or gRPC. Everyone thought this would fundamentally change the landscape, and it did--up to an extent. However, a new set of challenges emerged. What about cross-services communication? What about observability between microservices, such as logging or tracing? What about metrics?
+We all know that microservice architecture are the perfect fit for cloud native applications and it increases the delivery velocities greatly.   Envision you have many microservices that are delivered by multiple team, how do you observe the the overall platform and each of the service to find out exactly what is going on with each of the service?  When something goes wrong, how do you know which service or which communication among the few services are causing the problem?
```

Comment:
microservice architecture _is_ the perfect
delivered by multiple _teams_
overall platform and each of the _services_